examples/Getting Started with Boutiques.ipynb

```diff
@@ -0,0 +1,351 @@
+    "```\n",
+    "{\n",
+    "    \"id\" : \"variational_params_group\",\n",
+    "    \"name\" : \"Variations on Default Functionality\",\n",
+    "    \"description\" : \"Mutually exclusive options that specify variations on how BET should be run.\",\n",
+    "    \"members\" : [\"robust_iters_flag\", \"residual_optic_cleanup_flag\", \"reduce_bias_flag\", \"slice_padding_flag\", \"whole_set_mask_flag\", \"additional_surfaces_flag\", \"additional_surfaces_t2\"],\n",
+    "    \"mutually-exclusive\" : true\n",
+    "}\n",
+    "```\n",
+    "\n",
+    "Though an example of `one-is-required` input groups is not available in our BET example, you can investiage a validated identifier [here](https://github.com/neurodata/boutiques-tools/blob/master/cbrain_task_descriptors/ndmg.json#L13) to see how it is implemented."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Step 3.3: (optional) Extending the identifier\n",
+    "\n",
+    "Now that the basic implementation of this tool has been done, you can check the [documentation](http://boutiques.github.io/docs) to explore deeper functionality of Boutiques. For example, if you have created a Docker or Singularity container, you can associate an image with your identifier and any compute resource with Docker or Singularity installed will launch the executable through them (an example of using Docker can be found [here](https://github.com/neurodata/boutiques-tools/blob/master/cbrain_task_descriptors/ndmg.json#L6))."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 4: Validating the identifier\n",
+    "\n",
+    "Once you've completed your Boutiques identifier, you should run the [validator](https://github.com/boutiques/boutiques/blob/master/tools/validator.rb) to ensure that you have created it correctly. The `README.md` [here](https://github.com/boutiques/boutiques/tree/master/tools) describes how to install and use the validator on your identifier."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 5: Using the identifier\n",
+    "\n",
+    "Once the identifier has been validated, you can then deploy your tool for testing. You can use the `localExec.py` tool described [here](https://github.com/boutiques/boutiques/tree/master/tools) to launch your container locally for preliminary testing. Once you feel comfortable with your tool, you can contact your system administrator and have them integrate it into their compute resources so you can test and use it to process your data."
```

Comment:
We tend to call `manifest` what you refer as `identifier`. If `manifest` sounds too jargonic, maybe `tool descriptor` would be better.